### DISCUSSION TITLE
Can't open PHPMYADMIN

### DISCUSSION THREAD

**Discussion by dthief55** [leokhoa/laragon#525] [General]
Why i cant open phpMyAdmin from laragon, when i click "phpMyAdmin" form Laragon i just directed to this site

**Comment by Pen-y-Fan** [comment]
See #516 and #393

**Reply by dthief55** [reply]
> See #516 and #393 

tgx its done

**Comment by ShivamKushwahCoding** [comment]
Make sure you have installed "phpMyAdmin" by going to "Menu/Tools/Quick Add/phpMyAdmin"
And Also Make sure you have installed "mysql" by going to "Menu/Tools/Quick Add/mysql"

This worked for me.

**Reply by robisetiawan** [reply]
thanks, it worked for me

**Reply by abolfazlsoufi** [reply]
It worked for me, thanks

**Reply by web-v2** [reply]
Excelente muchas gracias!

**Reply by web-v2** [reply]
Luego de agregar phpmyadmin y tratar de acceder me sale este error en el navegador:
`Fatal error: Unparenthesized a ? b : c ? d : eis not supported. Use either(a ? b : c) ? d : eora ? b : (c ? d : e) in D:\APPs\laragon\etc\apps\phpMyAdmin\libraries\classes\DatabaseInterface.php on line 615`

**Reply by web-v2** [reply]
No me funciono ayuda...

**Reply by dawoodPK** [reply]
Thanks! It worked for me.

**Comment by dthief55** [comment]
Thankyou for the help, it works for mine

**Comment by web-v2** [comment]
Hola,
Si utilizas Laragon debes descargar el paquete de phpMyAdmin en la web, es un archivo .zip, lo pegas y descomprimes en la siguiente ruta: C:\laragon\etc\apps
Gracias
